querydsl-core/src/main/java/com/querydsl/core/types/ConstantImpl.java

```diff
@@ -28,50 +28,52 @@
 
     private static final int CACHE_SIZE = 256;
 
-    @SuppressWarnings({"rawtypes", "unchecked"}) //generic array creation not possible
-    private static final Constant<Character>[] CHARACTERS = new Constant[CACHE_SIZE];
```

Comment:
No it's fine, I just thought the diff would be a little more cleaner.
Since there is a new line introduced, git mismatches the lines a bit.